docs/topics/whatsnew16.md

```diff
@@ -667,6 +667,20 @@ It works with all language constructs, including inline functions.
 
 Learn more about Kover on its [GitHub repository](https://github.com/Kotlin/kotlinx-kover).
 
+## Coroutines 1.6.0-RC
+
+`kotlinx.coroutines` [1.6.0-RC](https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.6.0-RC) is out with 
+multiple features and improvements:
+
+* [New Kotlin/Native memory manager](#preview-of-the-new-memory-manager) support
```

Comment:
```suggestion
* Support for the [new Kotlin/Native memory manager](#preview-of-the-new-memory-manager)
```

more than three nouns in a row are hard to parse.

Reply:
Thank you!